portfolio/src/main/webapp/index.html

```diff
@@ -10,48 +10,46 @@
             background-image: url(/images/background.jpg);
             background-size: 400% 1390%;
         }
-        bingbound{
```

Comment:
Please move these style blocks into a .css file.  And also pay attention to add an empty line between each style block

Reply:
Done